src/Symfony/Component/Validator/Mapping/Loader/XmlFileLoader.php

```diff
@@ -185,8 +185,8 @@ protected function parseFile($path)
     /**
      * Loads the validation metadata from the given XML class description.
      *
-     * @param ClassMetadata $metadata         The metadata to load
-     * @param array         $classDescription The XML class description
+     * @param ClassMetadata     $metadata         The metadata to load
+     * @param \SimpleXMLElement $classDescription The XML class description
```

Comment:
what about completely removing the doc block and adding the `\SimpleXMLElement` type hint to the signature itself?